nixos/modules/security/acme.nix

```diff
@@ -235,6 +243,7 @@ let
 
         StateDirectory = "acme/${cert}";
 
+        ReadWritePaths = commonServiceConfig.ReadWritePaths ++ webroots;
```

Comment:
I wonder: Does the selfsign service really need access to the webroot? What for?

Reply:
Indeed, looking at the script, I don't think we need a write access for a selfsigned renewal.

> To solve the second point all you really have to do is add the webroots only in serviceConfig around line 243

^ line 243 being the self-signed certificate service, it lead me to think I was missing something.

That being said, we might both missing something here :smile: 

@m1cr0man did you mistakenly pointed to the self-signed renewal part instead of the acme part or are we actually missing something?